tests/verifyRemoteInstances_test.js

```diff
@@ -1,9 +1,15 @@
 const assert = require('assert');
-const page = require('./pages/remoteInstancesPage');
+
+const { remoteInstancesPage, pmmInventoryPage } = inject();
 
 const instances = new DataTable(['name']);
+const remotePostgreSQL = new DataTable(['instanceName', 'trackingOption', 'checkAgent']);
+
+remotePostgreSQL.add(['doNotTrack', remoteInstancesPage.fields.doNotTrack, pmmInventoryPage.fields.postgresExporter]);
```

Comment:
`doNotTrack`, I am sorry but I can't understand this serviceName when I will see it in the test reports

Reply:
What about `doNotTrackTrackingOption` ? :)

Reply:
Or `postgresDoNotTrackTrackingOption` ? :)